woof-code/rootfs-skeleton/root/.xinitrc

```diff
@@ -55,6 +55,7 @@ if [ -e /usr/bin/wireplumber ]; then
         while [ ! -e /tmp/runtime-spot/pipewire-0 ]; do sleep 0.1; done
         run-as-spot pipewire-pulse &
         run-as-spot wireplumber &
+	run-as-spot pipewire-media-session &
```

Comment:
Isn't WirePlumber an alternative to pipewire-media-session?

Reply:
(btw, Bluetooth audio and the audio jack work just fine for me, with WirePlumber and without pipewire-media-session)

Reply:
@dimkr 
Thanks for clarification I will close this PR and release new PR to avoid confusion